app/javascript/vue/BookShelf.vue

```diff
@@ -1,18 +1,26 @@
 <template>
   <div>
-    <book-card v-for="book in books" :key="book.id" :book="book"/>
+    <book-card v-for="book in booksLocal" :key="book.id" :book="book"/>
   </div>
 </template>
 
 <script>
   import BookCard from './BookCard'
+  import booksData from './books-data'
 
   export default {
-    props: {
-      books: Array
+    data () {
+      return {
+        booksLocal: this.books
```

Comment:
@isabloCh Forgot about this one. I think you shouldn't set it to a prop if later on you are set value from `booksData.getAll()`. If I miss something then please let me know